### PR TITLE
Add google cloud example file

### PR DESCRIPTION
**Pull request by Georgehe4**

### CODE REVIEW

**Comment by akmorrow13** [comment]
move to notebooks folder and rename to gce-1000genomes

**Comment by Georgehe4** [comment]
@akmorrow13 done.

File: example-files/notebooks/gce-1000genomes.ipynb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "6dc8eba5981a49f7b3c3c0dd39a5f23f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "<p>Failed to display Jupyter Widget of type <code>Reads</code>.</p>\n",
+       "<p>\n",
```
Comment (akmorrow13) [inline example-files/notebooks/gce-1000genomes.ipynb]: Could you clear cell output so we don't push the error message?

**Comment by Georgehe4** [comment]
updated

**Pull request merged**